README.md

```diff
@@ -16,22 +12,28 @@ don't plan on using within our own apps, we will be actively fixing bugs as they
 
 ## Example
 
-To run the example project, clone the repo, and run `pod install` from the Example directory first.  Then build and run the 
-TTCalendarPicker-Example project.
+To run the example project, clone the repo then build and run the TTCalendarPicker-Example project.
 
 ## Requirements
 
-- iOS 10.0+
-- Xcode 10.0+
-- Swift 4.0+
+- iOS 13.0+
```

Comment:
Does this really work back on iOS 13 ?

Reply:
yes, the minimum deployment target in project is ios13